app/controllers/authenticated_controller.rb

```diff
@@ -0,0 +1,20 @@
+class AuthenticatedController < ApplicationController
+  include Pundit
+  include GDS::SSO::ControllerMethods
+
+  protect_from_forgery
+
+  prepend_before_action :authenticate_user!
+  # before_action :require_signin_permission!
+
+  unless Rails.env.test?
```

Comment:
Could we get rid of this condition, you think? The unit tests should be able to avoid this block if needed, by using stubs.